app/src/main/java/com/doctor/yumyum/presentation/ui/main/MainActivity.kt

```diff
@@ -0,0 +1,56 @@
+package com.doctor.yumyum.presentation.ui.main
+
+import android.os.Bundle
+import androidx.lifecycle.ViewModelProvider
+import com.doctor.yumyum.R
+import com.doctor.yumyum.databinding.ActivityMainBinding
+import com.doctor.yumyum.common.base.BaseActivity
+import com.doctor.yumyum.presentation.viewmodel.MainViewModel
+import com.google.android.material.bottomnavigation.BottomNavigationView
+
+class MainActivity : BaseActivity<ActivityMainBinding>(R.layout.activity_main) {
+
+    private lateinit var vm: MainViewModel
```

Comment:
```suggestion
    val viewModel by viewModels<MainViewModel>()
```
보일러플레이트 제거겸 이번에 fragment ktx 써보는거 어떨까용?
https://developer.android.com/kotlin/ktx?hl=ko#fragment